---
description: "詳細: '<name1>' は '<name2>' から継承するため、'Catch' ブロックに到達することはありません"
title: "'<name1>' は '<name2>' から継承するため、'Catch' ブロックに到達することはありません"
ms.date: 07/20/2015
f1_keywords:
- BC42029
- vbc42029
helpviewer_keywords:
- BC42029
ms.assetid: 4bc84a84-ec7e-49db-8794-4fb71d36d5c0
ms.openlocfilehash: 9fbbad2d11f261f68ad1e4f63cb691805d59995e
ms.sourcegitcommit: 10e719780594efc781b15295e499c66f316068b8
ms.translationtype: HT
ms.contentlocale: ja-JP
ms.lasthandoff: 02/14/2021
ms.locfileid: "100422758"
---
# <a name="catch-block-never-reached-because-name1-inherits-from-name2"></a><span data-ttu-id="bd2ff-103">'\<name1>' は '\<name2>' から継承するため、'Catch' ブロックに到達することはありません</span><span class="sxs-lookup"><span data-stu-id="bd2ff-103">'Catch' block never reached, because '\<name1>' inherits from '\<name2>'</span></span>

<span data-ttu-id="bd2ff-104">コード内の `Catch` ブロックに到達することはありません。</span><span class="sxs-lookup"><span data-stu-id="bd2ff-104">A `Catch` block within the code is never reached.</span></span>  
  
 <span data-ttu-id="bd2ff-105">既定では、このメッセージは警告です。</span><span class="sxs-lookup"><span data-stu-id="bd2ff-105">By default, this message is a warning.</span></span> <span data-ttu-id="bd2ff-106">警告を非表示にする方法や、警告をエラーとして扱う方法の詳細については、「 [Configuring Warnings in Visual Basic](/visualstudio/ide/configuring-warnings-in-visual-basic)」を参照してください。</span><span class="sxs-lookup"><span data-stu-id="bd2ff-106">For more information on hiding warnings or treating warnings as errors, see [Configuring Warnings in Visual Basic](/visualstudio/ide/configuring-warnings-in-visual-basic).</span></span>  
  
 <span data-ttu-id="bd2ff-107">**エラー ID:** BC42029</span><span class="sxs-lookup"><span data-stu-id="bd2ff-107">**Error ID:** BC42029</span></span>  
  
## <a name="to-correct-this-error"></a><span data-ttu-id="bd2ff-108">このエラーを解決するには</span><span class="sxs-lookup"><span data-stu-id="bd2ff-108">To correct this error</span></span>  
  
- <span data-ttu-id="bd2ff-109">余分な `Catch` ブロックを削除します。</span><span class="sxs-lookup"><span data-stu-id="bd2ff-109">Remove the extraneous `Catch` block.</span></span>  
  
## <a name="see-also"></a><span data-ttu-id="bd2ff-110">関連項目</span><span class="sxs-lookup"><span data-stu-id="bd2ff-110">See also</span></span>

- [<span data-ttu-id="bd2ff-111">Try ブロックと Catch ブロックを使用して例外をキャッチする方法</span><span class="sxs-lookup"><span data-stu-id="bd2ff-111">How to use the try/catch block to catch exceptions</span></span>](../../standard/exceptions/how-to-use-the-try-catch-block-to-catch-exceptions.md)
- [<span data-ttu-id="bd2ff-112">Try...Catch...Finally ステートメント</span><span class="sxs-lookup"><span data-stu-id="bd2ff-112">Try...Catch...Finally Statement</span></span>](../language-reference/statements/try-catch-finally-statement.md)
